### PR TITLE
Closes #2308 #2198 - Rewrite about_methods and about_properties for c…

### LINKED ISSUE #2308: Amend "about_Methods" to contrast the syntax differences with argument mode, show pitfalls
<sup>Follow-up from a discussion at https://github.com/PowerShell/PowerShell/issues/6576.</sup>

Methods calls are like command/function calls at a certain level of abstraction, so users can get confused between the distinct syntax requirements:

From the perspective of a shell user mostly familiar with command syntax (argument mode), the tricky differences are:

* parentheses are required (vs. whitespace in argument mode)
* arguments must be `,`-separated (vs. whitespace in argument mode; in argument mode,  `,` constructs arrays)
* bareword arguments (unquoted strings) aren't allowed

Conversely, for someone coming from the developer world, calling your own helper functions such as `function foo ($bar, $baz) { ... }` as `foo('one', 'two')` is a real temptation, with a rude awakening.

I suggest amending [about_Methods](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Methods.md) as follows:

* Mentioning these pitfalls.

* Also making it clear that method calls are limited to _expression mode_ and are essentially a 
 pass-through feature to the realm PowerShell is _built on_, the .NET framework, which is a realm with _different rules and syntax_.  
Add a reference to `about_Parsing`

On a side note, I also suggest revise the following confusing sentence re member enumeration which is present not only in `about_Methods`, but also `about_Properties`:

> If the method [/ property] exists on the individual objects and on the collection, PowerShell does not alter the result.

This should read something like:

> If the method / property exists on both the individual objects and on the collection itself, it is (only) the collection's member that is invoked.

To illustrate this with a concrete example:

```powershell
PS> (@{ length = 'foo' }, @{ length = 'bar'}).length
2   # Collection-level `.Length` property took precedence
```

Version(s) of document impacted
------------------------------
- [x] Impacts 6.1 document
- [x] Impacts 6.0 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### PR DESCRIPTION
**Pull request by bobbytreed**
…larification

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [X] Impacts 6.0 document
- [X] Impacts 5.1 document
- [X] Impacts 5.0 document
- [X] Impacts 4.0 document
- [X] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-3.0&branch=pr-en-us-2840) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-3.0&branch=pr-en-us-2840) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-4.0&branch=pr-en-us-2840) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-4.0&branch=pr-en-us-2840) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-5.0&branch=pr-en-us-2840) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-5.0&branch=pr-en-us-2840) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.1/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-5.1&branch=pr-en-us-2840) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-5.1&branch=pr-en-us-2840) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/6/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-6&branch=pr-en-us-2840) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/6/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-6&branch=pr-en-us-2840) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.1/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/6/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/6/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C21%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201808211640192376-2840%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=W65HafL%2FwEz1EwlzaATxWR%2B%2FW0H4lYF83Uq%2BeQeMjS4%3D&st=2018-08-21T16%3A51%3A14Z&se=2018-09-21T16%3A56%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md [hunk 1]
```diff
@@ -106,19 +114,27 @@ two method signatures:
 ```
 
 The first method signature takes the destination file name (and a path). The
-following example use The first `CopyTo` method to copy the `Final.txt` file to
+following example uses The first `CopyTo` method to copy the `Final.txt` file to
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md]: Fix the capitalization - "uses The"

**Comment by bobbytreed** [comment]
#sign-off

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md [hunk 2]
```diff
@@ -124,10 +130,10 @@ For example, the following command displays the values of all the properties
 of the PowerShell.exe file.
 
 ```powershell
-PS> Get-ChildItem $pshome\PowerShell.exe | Format-List -Property *
+Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md]: Why lowercase '-property'?

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md [hunk 3]
```diff
@@ -106,19 +114,27 @@ two method signatures:
 ```
 
 The first method signature takes the destination file name (and a path). The
-following example use The first `CopyTo` method to copy the `Final.txt` file to
+following example uses The first `CopyTo` method to copy the `Final.txt` file to
```
Comment (sdwheeler) [inline reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md]: Fix the capitalization - "uses The"

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md [hunk 4]
```diff
@@ -106,19 +114,27 @@ two method signatures:
 ```
 
 The first method signature takes the destination file name (and a path). The
-following example use The first `CopyTo` method to copy the `Final.txt` file to
+following example uses The first `CopyTo` method to copy the `Final.txt` file to
```
Comment (sdwheeler) [inline reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md]: Fix the capitalization - "uses The"

File: reference/6/Microsoft.PowerShell.Core/About/about_Methods.md [hunk 5]
```diff
@@ -106,19 +114,27 @@ two method signatures:
 ```
 
 The first method signature takes the destination file name (and a path). The
-following example use The first `CopyTo` method to copy the `Final.txt` file to
+following example uses The first `CopyTo` method to copy the `Final.txt` file to
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_Methods.md]: Fix the capitalization - "uses The"

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md [hunk 6]
```diff
@@ -125,10 +130,10 @@ For example, the following command displays the values of all the properties
 of the PowerShell.exe file.
 
 ```powershell
-C:\PS> Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
+Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
```
Comment (sdwheeler) [inline reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md]: Why lowercase '-property'?

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md [hunk 7]
```diff
@@ -125,10 +130,10 @@ For example, the following command displays the values of all the properties
 of the PowerShell.exe file.
 
 ```powershell
-C:\PS> Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
+Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
```
Comment (sdwheeler) [inline reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md]: Why lowercase '-property'?

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md [hunk 8]
```diff
@@ -125,10 +130,10 @@ For example, the following command displays the values of all the properties
 of the PowerShell.exe file.
 
 ```powershell
-C:\PS> Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
+Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md]: Why lowercase '-property'?

File: reference/6/Microsoft.PowerShell.Core/About/about_Properties.md [hunk 9]
```diff
@@ -125,10 +130,10 @@ For example, the following command displays the values of all the properties
 of the PowerShell.exe file.
 
 ```powershell
-C:\PS> Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
+Get-ChildItem $pshome\PowerShell.exe | Format-List -property *
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_Properties.md]: Why lowercase '-property'?

Comment (bobbytreed) [reply reference/6/Microsoft.PowerShell.Core/About/about_Properties.md]: Addressed

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md (hunk 8 above)
Comment (bobbytreed) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md]: Addressed

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md (hunk 7 above)
Comment (bobbytreed) [reply reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md]: Addressed

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md (hunk 6 above)
Comment (bobbytreed) [reply reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md]: Addressed

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md (hunk 4 above)
Comment (bobbytreed) [reply reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md]: Addressed

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md (hunk 3 above)
Comment (bobbytreed) [reply reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md]: Addressed

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md (hunk 2 above)
Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md]: Addressed

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md (hunk 1 above)
Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md]: Addressed

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-3.0&branch=pr-en-us-2840) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-3.0&branch=pr-en-us-2840) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-4.0&branch=pr-en-us-2840) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-4.0&branch=pr-en-us-2840) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-5.0&branch=pr-en-us-2840) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-5.0&branch=pr-en-us-2840) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.1/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-5.1&branch=pr-en-us-2840) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-5.1&branch=pr-en-us-2840) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/6/Microsoft.PowerShell.Core/About/about_Methods.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_methods?view=powershell-6&branch=pr-en-us-2840) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/6/Microsoft.PowerShell.Core/About/about_Properties.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_properties?view=powershell-6&branch=pr-en-us-2840) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/3.0/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/3.0/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/4.0/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/4.0/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.0/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.0/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.1/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/5.1/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Methods.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Methods.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/6/Microsoft.PowerShell.Core/About/about_Methods.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Properties.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Properties.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue2308/reference/6/Microsoft.PowerShell.Core/About/about_Properties.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C21%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201808211745322201-2840%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=L1YndOgMj4213e28WVZAKauaGYU8i53tV4zQAGUW6ZM%3D&st=2018-08-21T17%3A55%3A57Z&se=2018-09-21T18%3A00%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bobbytreed** [comment]
#sign-off

**Comment by bobbytreed** [comment]
#sign-off

**Pull request merged**